documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -708,6 +701,10 @@ For a full list of possible `<country>` values, see the abbreviations used in th
 [[advanced-options-nonint]]
 === Advanced options
 
+Advanced options are intended for use during troubleshooting. Available options may differ between Raspberry Pi models.
```

Comment:
Suggestions:

* remove "may" from "may differ" -- since we make the software, I think it's safe to say that options _DO_ differ between models!
* the new introductory sentence sounds a bit clunky and isn't quite right; there are a lot of advanced options users might use for purposes other than troubleshooting. unfortunately i don't have a better suggestion, but I'm open to ideas. Even if those ideas are 'remove this entire paragraph' :-)

If we borrowed that introductory sentence from elsewhere in the docs, I think we should consider rewriting it in its original home as well. Unless it's more true in that setting?

Reply:
```suggestion
```